content/swift/concepts/collections/collections.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: 'Collections'
+Description: 'Collections are data structures used to store multiple values in a given variable.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Collections'
+  - 'Arrays'
+  - 'Dictionaries'
+  - 'Sets'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**Collections** are [data structures](https://www.codecademy.com/resources/docs/general/data-structures) that store multiple values of the same, or different, [type(s)](https://www.codecademy.com/resources/docs/swift/data-types).
+
+Swift provides these collection types:
+
+- [arrays](https://www.codecademy.com/resources/docs/swift/arrays)
+- [sets](https://www.codecademy.com/resources/docs/swift/sets)
+- [dictionaries](https://www.codecademy.com/resources/docs/swift/dictionaries)
+
+Collections are value types, whose instances are copied when they are assigned to a variable or constant, or when they are passed to a function.
+
+## Syntax
+
+In Swift, collections are defined using different syntaxes, depending on the type of collection:
+
+```swift
+var fruits = ["apple", "banana", "orange"] // An array
+
+var colors = Set("red", "green", "blue") // A set
+
+var scores = ["Alice": 90, "Bob": 80, "Charlie": 70] // A dictionary
+```
```

Comment:
Can we turn section to show the Collection protocol? And here, we can explain that adhere to Collection allows access to certain operations